src/fastboot-info.js

```diff
@@ -7,13 +7,15 @@ var FastBootResponse = require('./fastboot-response');
  * current HTTP request from FastBoot. This is injected
  * on to the FastBoot service.
  */
-function FastBootInfo(request, response, hostWhitelist) {
+function FastBootInfo(request, response, infoOptions) {
   this.deferredPromise = RSVP.resolve();
+  const { hostWhitelist, metaData } = infoOptions;
```

Comment:
Feel free to disregard, but this is the only `const` here and I've generally seen us follow the "liberal-let" ([from here](http://madhatted.com/2016/1/25/let-it-be)), this might want to be a `let`?